### ISSUE TITLE
Sprites.css references githubusercontent.com

### ISSUE THREAD

**Issue by ajmas** [EducationLink/vue-tel-input#31]
Looking at sprites.css, I see the flag images are loaded from githubusercontent.com. This is a potential issue.

There should be a way for referencing these image files locally or at least provided a method to specify an alternative base path for these images.

Code in question:

```css
.iti-flag {
  width: 20px;
  height: 15px;
  box-shadow: 0px 0px 1px 0px #888;
  background-image: url("https://raw.githubusercontent.com/EducationLink/vue-tel-input/master/src/assets/flags.png");
  background-repeat: no-repeat;
  background-color: #dbdbdb;
  background-position: 20px 0;
}
@media only screen and (-webkit-min-device-pixel-ratio: 2), only screen and (min--moz-device-pixel-ratio: 2), only screen and (-o-min-device-pixel-ratio: 2), only screen and (min-device-pixel-ratio: 2), only screen and (min-resolution: 192dpi), only screen and (min-resolution: 2dppx) {
  .iti-flag {
    background-image: url("https://raw.githubusercontent.com/EducationLink/vue-tel-input/master/src/assets/flags@2x.png");
  }
}
```

**Comment by iamstevendao** [comment]
@ajmas Thanks for pointing it out, this was a temporary fix for #24. However, we still don't have time to work on a proper one. Any contributions are really appreciated!

**Comment by robbeman** [comment]
Loading images from a different source does not make them smaller, so why not just include the image from a relative path as it was before? (If the reason for doing this is that github is gzipping and caching stuff by default then people should just configure their own server like that.)
![image](https://user-images.githubusercontent.com/3245659/46861960-26c65500-ce14-11e8-9931-8589587c7507.png)

Because now the image somehow became unavailable to me (just me?). If the image changes location in the repo the picker will become broken on any site using it, no?
![image](https://user-images.githubusercontent.com/3245659/46861882-fb436a80-ce13-11e8-9bb0-f71415cd200a.png)

Also: are you allowed to use github as a CDN?

**Comment by iamstevendao** [comment]
Fixed in v2.0.10, thank to @garrensweet
